bin/celery-helper.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python -i
```

Comment:
Any reason for not having this as a runner command to reuse the sentry initiation? e.g. https://github.com/getsentry/sentry/blob/master/src/sentry/runner/commands/queues.py

Reply:
This is very experimental for now, so still early to promote this to a runner command